.github/workflows/ci_pr.yaml

```diff
@@ -47,6 +47,9 @@ jobs:
       timeout-minutes: 20
       run: |
         golint ./... > golint.err 2>&1
+        if ! [ -s golint.err ]; then
+          exit 0
+        fi
```

Comment:
Should the CI pipeline run the terraform unit tests?

Reply:
Yes, sorry it was not committed when you reviewed it. A job for unit tests have been added here: https://github.com/GoogleCloudPlatform/anthos-samples/pull/26/files#diff-4becefff464f0184f0b2863fc9b24cf738be50aa1014a3c53483476e8027fc90R71-R93

I will add a README about running the unit tests for anyone using the repository

Reply:
I've updated the contributing section in the [main README here](https://github.com/GoogleCloudPlatform/anthos-samples/tree/unit-tests)

Also added a section [specific to this sample here](https://github.com/GoogleCloudPlatform/anthos-samples/tree/unit-tests)

The expectation is have access atleast on Google Cloud Project to run the test against.